package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "loopback-workspace",
-  "version": "5.2.1",
+  "version": "6.0.0",
```

We usually set the version number as part of the release process. Let's use a pre-release version for this PR.

```suggestion
  "version": "6.0.0-dev",
```

---

@bajtos good catch, I realized apiconnect-cli-loopback depends on 4.x, not 5.x, so we have to patch 4.x instead.